Tests/SwiftParserTest/translated/ErrorsTests.swift

```diff
@@ -279,7 +279,7 @@ final class ErrorsTests: ParserTestCase {
     assertParse(
       """
       func incompleteThrowType() {
-        let _: () 1️⃣throws
+        let _: ()1️⃣ 2️⃣throws
```

Comment:
I assume this had a second that was then removed after?

Reply:
You're right. Reverted this change.